test/support/flux_parser.ex

```diff
@@ -0,0 +1,150 @@
+defmodule TelemetryInfluxDB.Test.FluxParser do
+  alias NimbleCSV.RFC4180, as: CSV
+
+  @column_types %{
+    "boolean" => :boolean,
+    "double" => :double,
+    "string" => :string,
+    "long" => :long,
+    "unsignedLong" => :unsigned_long,
+    "dateTime:RFC3339" => :datetime
+  }
+
+  def parse_tables(csv) do
```

Comment:
Can we decompose it a bit? I mean move the body from those anonymous functions to some separate named functions. Having a name for a step would give a little bit more context and improve readibility :)